src/net/protocol_agnostic/msg.rs

```diff
@@ -0,0 +1,8 @@
+use priv_prelude::*;
+
+#[derive(Debug, Serialize, Deserialize)]
+pub struct PaRendezvousMsg {
```

Comment:
Why  not move `PaRendezvousMsg` to `protocol_agnostic/mod.rs`?
Currently, it's an 8 line file and it also has redundant labeling: `msg::PaRendezvousMsg`.